### PR TITLE
add mobile menu pop-up using DOM Manipulation

### PR DESCRIPTION
**Pull request by joyapisi**
# What We Did 🏋🏾‍♀️🏋🏾‍♂️:
`DOM Manipulation`
-Use JavaScript to manipulate DOM elements.
-Use JavaScript events

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi team,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

You have submitted a project, that has a templated text on your README, kindly remove it.
Screenshot 2023-03-06 at 10 46 38 PM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -39,7 +39,7 @@ nav {
   background-color: transparent;
 }
 
-.navigationList {
+#navigationList {
```
Comment (huseyinbiyik) [inline style.css:42]: Good job with the background blur, it looks fantastic 👌  But let's make sure when the user clicks on the menu link, the menu closes.
Screenshot 2023-03-06 at 11 43 38 PM
Screen Recording 2023-03-06 at 11 41 36 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- New linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝

## Required Changes ♻️
- Kindly add some space before the header like on the Figma template.
Screenshot 2023-03-06 at 11 47 11 PM
Screenshot 2023-03-06 at 11 47 35 PM

-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML"
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @TEAM      , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] linters are passing
- [x] Professional README 💯 
- [x] Nice UI design. 🥇 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

**Pull request merged**